views/js/previewer/plugins/content/scoringHighlighter.js

```diff
@@ -115,14 +136,19 @@ define([
              * @param {jQueryElement} $eraser 
              * @param {Boolean} isEraserOn 
              */
-            this.toggleEraser = ($eraser, isEraserOn) => {
-                if (isEraserOn) {
-                    $eraser.addClass('eraser-on');
-                    $(CONTAINER_SELECTOR + ' .' + CLASS_NAME).off('click').on('click', clearHighlightAndSave);
-                } else {
-                    $eraser.removeClass('eraser-on');
-                    $(CONTAINER_SELECTOR + ' .' + CLASS_NAME).off('click')
+            this.toggleEraser = ($eraser, turnOnEraser) => {
+                // Only turn on eraser if there are highlights
+                if (this.hasHighlights) {
+                    if (turnOnEraser) {
+                        $eraser.addClass('eraser-on');
+                        $(CONTAINER_SELECTOR + ' .' + CLASS_NAME).off('click').on('click', clearHighlightAndSave);
+                        $(CONTAINER_SELECTOR + ' .' + CLASS_NAME).addClass('erase');
```

Comment:
Do we need also to remove this class on turn eraser off?

Reply:
you are right, I missed it in all the chaos